test/general/openpgp.js

```diff
@@ -304,6 +304,37 @@ const twoPasswordGPGFail = ['-----BEGIN PGP MESSAGE-----',
 '=cHCV',
 '-----END PGP MESSAGE-----'].join('\n');
 
+const feng_msg_bad = `-----BEGIN PGP MESSAGE-----
```

Comment:
Maybe name these something more descriptive of the bug and less descriptive of feng 😂